### ISSUE TITLE
Docker build from source process fails

### ISSUE THREAD

**Issue by mboersma** [Azure/aks-engine#97]
**Is this a request for help?**:

---

**Is this an ISSUE or FEATURE REQUEST?** (choose one):

---

**What version of aks-engine?**:

---

**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm)**

**What happened**:

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

**Comment by elutsky** [comment]
Hi
I am completely new for GO, so maybe doing something wrong - sorry about this.
I am trying to build the aks-engine on my Windows laptop.
I cloned the repo, and did try to run the script ([as mentioned here](https://github.com/Azure/aks-engine/blob/master/docs/aksengine.md#docker-development-environment))

Expected: `make build ` pass
Actual: `make build` failed with error

> Makefile:57: recipe for target 'generate' failed
> make: *** [generate] Error 1
> 

Additional error details:

> gen.go:172: Could not locate package: cannot find package "golang.org/x/text" in any of:
>         /usr/local/go/src/golang.org/x/text (from $GOROOT)
>         /gopath/src/golang.org/x/text (from $GOPATH)
> exit status 1

My laptop GOPATH:
> set GOPATH=C:\WCDRepos\GO

In my computer the package seems to be exists in the specified folder:
![image](https://user-images.githubusercontent.com/7150258/50052470-29ec0300-012d-11e9-8213-48d128d405c1.png)

The issue, from my understanding, is that during build GO is looking for the _/x/text_ package in the _/gopath/src/**golang.org**_ folder, which is not mounted when running the container:

```
$pwd = (Get-Location).Path

docker build --pull -t aks-engine .
docker run --security-opt seccomp:unconfined -it `
	-v ${pwd}:/gopath/src/github.com/Azure/aks-engine `
	-w /gopath/src/github.com/Azure/aks-engine `
		aks-engine /bin/bash
```

When I did try to add the mount to the missing folder, it seems it helped with this error, but other errors occured:

```
$pwd = (Get-Location).Path

docker build --pull -t aks-engine .
docker run --security-opt seccomp:unconfined -it `
	-v ${pwd}:/gopath/src/github.com/Azure/aks-engine `
	**-v C:/WCDRepos/GO/Src/golang.org:/gopath/src/golang.org `**
	-w /gopath/src/github.com/Azure/aks-engine `
		aks-engine /bin/bash

```

> vendor/github.com/Azure/aks-engine/pkg/engine/fileloader.go
**> bindata: Failed to stat input path '../../parts': lstat ../../parts: no such file or directory**
> vendor/github.com/Azure/aks-engine/pkg/engine/fileloader.go:6: running "go-bindata": exit status 1
> ...
> vendor/github.com/Azure/aks-engine/pkg/i18n/const.go
> vendor/github.com/Azure/aks-engine/pkg/i18n/i18n.go
> vendor/github.com/Azure/aks-engine/pkg/i18n/resourceloader.go
**> bindata: Failed to stat input path '../../translations': lstat ../../translations: no such file or directory**

Am I doing something wrong?
What should be done to fix it?

Thanks!

**Comment by bacongobbler** [comment]
@elutsky looks like that should be fixed with #219. Let me know how that goes for you.
